docs/source/conf.py

```diff
@@ -188,6 +188,9 @@
     "display_version": True,
 }
 
+# Adding favicon to the docs
```

Comment:
Very minor nitpick for me that you don't need to worry about, @mrbazzan :) For comments that are their own line we try to have them always be a complete sentence with a period at the end. If it's an inline comment then we don't capitalize the first word or include a period. These are just the habits that I picked up from looking into Python coding standards :)

I'll fix this!